recursio-client/Managers/CharacterManager.gd

```diff
@@ -206,6 +206,7 @@ func _on_game_phase_started() -> void:
 	_game_manager.toggle_capture_points(true)
 
 func _on_game_phase_stopped() -> void:
+	toggle_swapping(true)
```

Comment:
Wouldn't this fit slightly better in `_on_preparation_phase_started`? Just to make sure it's `true` when the prep phase starts in case there was no game phase end beforehand for some reason

Reply:
Moved to `_on_preperation_phase_started`

Reply:
thx!